presto-main/src/main/java/com/facebook/presto/sql/planner/LocalExecutionPlanner.java

```diff
@@ -1247,7 +1250,7 @@ public PhysicalOperation visitAggregation(AggregationNode node, LocalExecutionPl
             return planGroupByAggregation(
                     node,
                     source,
-                    spillEnabled,
```

Comment:
can remove the spillEnabled variable now since its unused

Reply:
@rschlussel : This variable is already removed. We now only pass isAggregationSpillEnabled flag along with isDistinctAggregationSpillEnabled and isOrderByAggregationSpillEnabled.

Reply:
sorry got it confused due to collapsed code in github.  Looks good.